### PR TITLE
(CSM 1.0) CASMHMS-5163: Add procedures to change default and device specific credentials for River hardware.

### PR DESCRIPTION
**Pull request by rsjostrand-hpe**
Add procedures to change default and device specific credentials for River hardware.
- Update default Air-Cooled BMC and Leaf switch SNMP credentials
   - Tested on Mug 
- Change Air-Cooled Node BMC Credentials
   - Tested on Loki
- Change SMNP credentials on Leaf switches
   - Tested on Loki (Aruba) & Mug (Dell)
- Update default ServerTech PDU credentials used by the Redfish Translation Service
   - Tested on Mug
- Change credentials on ServerTech PDU
   - Tested on Gamora.

### CODE REVIEW

File: operations/security_and_authentication/Change_Air-Cooled_Node_BMC_Credentials.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Change Air-Cooled Node BMC Credentials
+
+This procedure will use the System Configuration Service (SCSD) to change all air-cooled Node BMCs in the system to the same global credential. 
+
+**Limitation:** All air-cooled and liquid cooled BMCs should share the same global credentials. The air-cooled Slingshot Switch Controllers (Router BMCs) must have the same credentials as the liquid-cooled Slingshot switch controllers.
```
Comment (nrockershousen) [inline operations/security_and_authentication/Change_Air-Cooled_Node_BMC_Credentials.md:5]: This should be a separate header and the text requires a couple of consistency edits. See below.

### Limitations 

All air-cooled and liquid-cooled BMCs share the same global credentials. The air-cooled Slingshot switch controllers (Router BMCs) must have the same credentials as the liquid-cooled Slingshot switch controllers.

File: operations/security_and_authentication/Change_Air-Cooled_Node_BMC_Credentials.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+### Prerequisites
+
+-   The Cray command line interface \(CLI\) tool is initialized and configured on the system.
+-   Review procedures for SCSD in the [Cray System Management (CSM) Administration Guide](../index.md#system-configuration-service)
+
+### Procedure
+
+1.  Create a SCSD payload file to change all Air-Cooled Node BMCs to the same global credential:
```
Comment (nrockershousen) [inline operations/security_and_authentication/Change_Air-Cooled_Node_BMC_Credentials.md:14]: "Air-Cooled Node" -> "air-cooled node"

File: operations/security_and_authentication/Change_SMNP_credentials_on_Leaf_switches.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
+# Change SMNP credentials on Leaf switches
```
Comment (nrockershousen) [inline operations/security_and_authentication/Change_SMNP_credentials_on_Leaf_switches.md:1]: Capitalization change: "Change SMNP Credentials on Leaf Switches"

File: operations/security_and_authentication/Change_SMNP_credentials_on_Leaf_switches.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+This procedure changes the SNMP credentials on management leaf switches in the system. Either a single leaf switch can be updated to use new SNMP credentials, or update all leaf switches in the system to use the same global SNMP credentials.
+
+**Note**: This procedure will not update the default SNMP credentials used when new leaf switches are added to the system. To update the default SNMP credentials for new hardware follow the [Update default Air-Cooled BMC and Leaf switch SNMP credentials](Update_default_Air-Cooled_BMC_and_Leaf_switch_SNMP_credentials.md) procedure.
```
Comment (nrockershousen) [inline operations/security_and_authentication/Change_SMNP_credentials_on_Leaf_switches.md:4]: **Note**:  ->   **NOTE:**

Comment (nrockershousen) [inline operations/security_and_authentication/Change_SMNP_credentials_on_Leaf_switches.md:4]: Add comma after hardware in second sentence

File: operations/security_and_authentication/Change_SMNP_credentials_on_Leaf_switches.md [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+    ```
+
+    Sample output for a system with 2 Dell leaf switches:
+    ```
+    Xname       Brand  Alias
+    x3000c0w14  Dell   sw-leaf-002
+    x3000c0w13  Dell   sw-leaf-001
+    ```
+    
+2. Update SNMP credentials for the `testuser` user on each leaf switch in the system. The SNMP `testuser` user requires 2 password to be provided for the SNMP Authentication and Privacy protocol passwords. Both of these passwords must be 8 characters or longer. In the examples below `foobar01` is the new SNMP Authentication password, and `foobar02` is the new SNMP Privacy password.
```
Comment (nrockershousen) [inline operations/security_and_authentication/Change_SMNP_credentials_on_Leaf_switches.md:30]: Add comma after "In the examples below"

File: operations/security_and_authentication/Change_SMNP_credentials_on_Leaf_switches.md [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+    Sample output for a system with 2 Dell leaf switches:
+    ```
+    Xname       Brand  Alias
+    x3000c0w14  Dell   sw-leaf-002
+    x3000c0w13  Dell   sw-leaf-001
+    ```
+    
+2. Update SNMP credentials for the `testuser` user on each leaf switch in the system. The SNMP `testuser` user requires 2 password to be provided for the SNMP Authentication and Privacy protocol passwords. Both of these passwords must be 8 characters or longer. In the examples below `foobar01` is the new SNMP Authentication password, and `foobar02` is the new SNMP Privacy password. 
+
+    1.  Aruba leaf switch configuration:
```
Comment (nrockershousen) [inline operations/security_and_authentication/Change_SMNP_credentials_on_Leaf_switches.md:32]: Change to "Configure the Aruba leaf switch:"

File: operations/security_and_authentication/Change_SMNP_credentials_on_Leaf_switches.md [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+        ```bash
+        ncn-m001# ssh admin@sw-leaf-001
+        sw-leaf-001# configure terminal
+        sw-leaf-001(config)# snmpv3 user testuser auth md5 auth-pass plaintext foobar01 priv des priv-pass plaintext foobar02
+        sw-leaf-001(config)# exit
+        sw-leaf-001# write memory
+        sw-leaf-001# exit
+        ```
+
+    2.  Dell leaf switch configuration:
```
Comment (nrockershousen) [inline operations/security_and_authentication/Change_SMNP_credentials_on_Leaf_switches.md:42]: Change to "Configure the Dell leaf switch:"

File: operations/security_and_authentication/Change_SMNP_credentials_on_Leaf_switches.md [hunk 8]
```diff
@@ -0,0 +1,112 @@
...
+        for XNAME in $(cray sls search hardware list --type comptype_mgmt_switch --format json | jq -r .[].Xname); do
+            echo "Updating SNMP creds for $XNAME"
+            vault kv get secret/hms-creds/$XNAME | 
+                jq --arg SNMP_AUTH_PASS "$SNMP_AUTH_PASS" --arg SNMP_PRIV_PASS "$SNMP_PRIV_PASS" \
+                    '.data | .SNMPAuthPass=$SNMP_AUTH_PASS | .SNMPPrivPass=$SNMP_PRIV_PASS' |
+                vault kv put secret/hms-creds/$XNAME -
+        done
+        ``` 
+
+4.  Restart REDS to pickup the new SNMP credentials:
```
Comment (nrockershousen) [inline operations/security_and_authentication/Change_SMNP_credentials_on_Leaf_switches.md:81]: Need to expand REDS for first time usage in procedure:

Restart the River Endpoint Discovery Service (REDS) to pickup the new SNMP credentials.

File: operations/security_and_authentication/Change_SMNP_credentials_on_Leaf_switches.md [hunk 9]
```diff
@@ -0,0 +1,112 @@
...
+    ncn-m001# kubectl -n services rollout restart deployment cray-reds
+    ncn-m001# kubectl -n services rollout status deployment cray-reds
+    ```
+
+5.  Wait for REDS to initialize itself:
+    ```bash
+    ncn-m001# sleep 2m
+    ```
+
+6.  Verify REDS was able to communicate with the leaf switches with update credentials:
```
Comment (nrockershousen) [inline operations/security_and_authentication/Change_SMNP_credentials_on_Leaf_switches.md:93]: Change "with update credentials" to "with the updated credentials"

File: operations/security_and_authentication/Change_credentials_on_ServerTech_PDUs.md [hunk 10]
```diff
@@ -0,0 +1,133 @@
+# Change credentials on ServerTech PDUs
```
Comment (nrockershousen) [inline operations/security_and_authentication/Change_credentials_on_ServerTech_PDUs.md:1]: Capitalization change: "Change Credentials on ServerTech PDUs"

File: operations/security_and_authentication/Change_credentials_on_ServerTech_PDUs.md [hunk 11]
```diff
@@ -0,0 +1,133 @@
...
+This procedure changes password used by the `admn` user on ServerTech PDUs. Either a single PDU can be updated to a new credential, or update all ServerTech PDUs in the system to the same global credentials.
+
+**Note:** This procedure does not update the default credentials that RTS uses for new ServerTech PDUs added to a system. To change the default credentials follow the [Update default ServerTech PDU credentials used by the Redfish Translation Service](Update_default_ServerTech_PDU_credentials_used_by_the_Redfish_Translation_Service.md) procedure.
```
Comment (nrockershousen) [inline operations/security_and_authentication/Change_credentials_on_ServerTech_PDUs.md:4]: **Note**: -> **NOTE:**

Comment (nrockershousen) [inline operations/security_and_authentication/Change_credentials_on_ServerTech_PDUs.md:4]: Add comma after credentials in second sentence

File: operations/security_and_authentication/Change_credentials_on_ServerTech_PDUs.md [hunk 12]
```diff
@@ -0,0 +1,133 @@
...
+        HTTP/1.1 204 No Content
+        Content-Type: text/html
+        Transfer-Encoding: chunked
+        Server: ServerTech-AWS/v8.0p
+        Set-Cookie: C5=1883488164; path=/
+        Connection: close
+        Pragma: JAWS v1.01
+        ```
+
+    **Note**: After 5 minutes the previous credential should stop working, as the existing session timed out.
```
Comment (nrockershousen) [inline operations/security_and_authentication/Change_credentials_on_ServerTech_PDUs.md:84]: **Note**: -> **NOTE:**

File: operations/security_and_authentication/System_Security_and_Authentication.md [hunk 13]
```diff
@@ -35,6 +35,11 @@ All connections through the Istio ingress gateway require authentication with a
     -   [Change EX Liquid-Cooled Cabinet Global Default Password](Change_EX_Liquid-Cooled_Cabinet_Global_Default_Password.md)
     -   [Provisioning a Liquid-Cooled EX Cabinet CEC with Default Credentials](Provisioning_a_Liquid-Cooled_EX_Cabinet_CEC_with_Default_Credentials.md)
     -   [Updating the Liquid-Cooled EX Cabinet Default Credentials after a CEC Password Change](Updating_the_Liquid-Cooled_EX_Cabinet_Default_Credentials_after_a_CEC_Password_Change.md)
+    -   [Update default Air-Cooled BMC and Leaf switch SNMP credentials](Update_default_Air-Cooled_BMC_and_Leaf_switch_SNMP_credentials.md)
```
Comment (nrockershousen) [inline operations/security_and_authentication/System_Security_and_Authentication.md:38]: This entire TOC section can be removed. We are only tracking this in /operations/index.md now. It looks like this got added back in a previous PR by mistake.

File: operations/security_and_authentication/Update_default_Air-Cooled_BMC_and_Leaf_switch_SNMP_credentials.md [hunk 14]
```diff
@@ -0,0 +1,196 @@
+# Update default Air-Cooled BMC and Leaf switch SNMP credentials
+
+This procedure updates the default credentials used when new air-cooled hardware is discovered for the first time. This includes the default Redfish credentials used for new air-cooled NodeBMCs and Slingshot switch BMCs (RouterBMCs), and SNMP credentials for new management leaf switches.
+
+**Important:** After this procedure is completed **going forward all future air-cooled hardware** added to the system will be assumed to be already configured with the new global default credential when getting added to the system.
+
+**Note:** This procedure will not update the Redfish or SNMP credentials for existing air-cooled devices. To change the credentials on existing air-cooled hardware follow the [Change Air-Cooled Node BMC Credentials](Change_Air-Cooled_Node_BMC_Credentials.md) and [Change SMNP credentials on Leaf switches](Change_SMNP_credentials_on_Leaf_switches.md) procedures.
```
Comment (nrockershousen) [inline operations/security_and_authentication/Update_default_Air-Cooled_BMC_and_Leaf_switch_SNMP_credentials.md:7]: **Note**: -> **NOTE:**

File: operations/security_and_authentication/Update_default_Air-Cooled_BMC_and_Leaf_switch_SNMP_credentials.md [hunk 15]
```diff
@@ -0,0 +1,196 @@
+# Update default Air-Cooled BMC and Leaf switch SNMP credentials
```
Comment (nrockershousen) [inline operations/security_and_authentication/Update_default_Air-Cooled_BMC_and_Leaf_switch_SNMP_credentials.md:1]: Capitalization change: "Update Default Air-Cooled BMC and Leaf Switch SNMP Credentials"

File: operations/security_and_authentication/Update_default_Air-Cooled_BMC_and_Leaf_switch_SNMP_credentials.md [hunk 16]
```diff
@@ -0,0 +1,196 @@
...
+
+**Important:** After this procedure is completed **going forward all future air-cooled hardware** added to the system will be assumed to be already configured with the new global default credential when getting added to the system.
+
+**Note:** This procedure will not update the Redfish or SNMP credentials for existing air-cooled devices. To change the credentials on existing air-cooled hardware follow the [Change Air-Cooled Node BMC Credentials](Change_Air-Cooled_Node_BMC_Credentials.md) and [Change SMNP credentials on Leaf switches](Change_SMNP_credentials_on_Leaf_switches.md) procedures. 
+
+## Prerequisites
+
+-   The Cray command line interface \(CLI\) tool is initialized and configured on the system.
+-   **Limitation:** The default global credentials used for liquid-cooled BMCs in the [Change Cray EX Liquid-Cooled Cabinet Global Default Password](Change_EX_Liquid-Cooled_Cabinet_Global_Default_Password.md) procedure needs to be the same as the one used in this procedure for air-cooled BMCs river hardware.
```
Comment (nrockershousen) [inline operations/security_and_authentication/Update_default_Air-Cooled_BMC_and_Leaf_switch_SNMP_credentials.md:12]: New section:

### Limitations

The default global credentials used for liquid-cooled BMCs in the [Change Cray EX Liquid-Cooled Cabinet Global Default Password](Change_EX_Liquid-Cooled_Cabinet_Global_Default_Password.md) procedure needs to be the same as the one used in this procedure for air-cooled BMCs river hardware.

File: operations/security_and_authentication/Update_default_Air-Cooled_BMC_and_Leaf_switch_SNMP_credentials.md [hunk 17]
```diff
@@ -0,0 +1,196 @@
...
+
+## Prerequisites
+
+-   The Cray command line interface \(CLI\) tool is initialized and configured on the system.
+-   **Limitation:** The default global credentials used for liquid-cooled BMCs in the [Change Cray EX Liquid-Cooled Cabinet Global Default Password](Change_EX_Liquid-Cooled_Cabinet_Global_Default_Password.md) procedure needs to be the same as the one used in this procedure for air-cooled BMCs river hardware. 
+
+## Procedure
+
+#### 1.1 Acquire site-init.
+Before redeploying REDS, update the `customizations.yaml` file in the `site-init` secret in the `loftsman` namespace.
```
Comment (nrockershousen) [inline operations/security_and_authentication/Update_default_Air-Cooled_BMC_and_Leaf_switch_SNMP_credentials.md:17]: Expand REDS to River Endpoint Discovery Service (REDS)

File: operations/security_and_authentication/Update_default_Air-Cooled_BMC_and_Leaf_switch_SNMP_credentials.md [hunk 18]
```diff
@@ -0,0 +1,196 @@
...
+        name: reds
+    spec:
+        charts:
+        - name: cray-hms-reds
+          version: $REDS_VERSION
+          namespace: services
+    EOF
+    ```
+
+2.  Merge customizations.yaml with reds-manifest.yaml
```
Comment (nrockershousen) [inline operations/security_and_authentication/Update_default_Air-Cooled_BMC_and_Leaf_switch_SNMP_credentials.md:152]: Missing punctuation at the end of this step

Comment (nrockershousen) [reply operations/security_and_authentication/Update_default_Air-Cooled_BMC_and_Leaf_switch_SNMP_credentials.md:152]: These file names are surrounded with backticks in text/steps above. This should be consistent.

File: operations/security_and_authentication/Change_credentials_on_ServerTech_PDUs.md [hunk 19]
```diff
@@ -0,0 +1,133 @@
...
+        ncn-m001# for PDU in $(cray hsm inventory redfishEndpoints list --type CabinetPDUController --format json | 
+          jq -r '.RedfishEndpoints[] | select(.FQDN | contains("rts")).ID'); do
+            echo "Updating password on $PDU"
+            vault kv get secret/pdu-creds/$PDU | 
+                jq --arg PASSWORD "$NEW_PDU_PASSWORD" '.data | .Password=$PASSWORD' |
+                vault kv put secret/pdu-creds/$PDU -
+        done
+        ```
+        
+6.  Restart RTS to pickup the new PDU credentials:
```
Comment (nrockershousen) [inline operations/security_and_authentication/Change_credentials_on_ServerTech_PDUs.md:112]: Redfish Translation Service (RTS)

File: operations/system_configuration_service/Set_BMC_Credentials.md [hunk 20]
```diff
@@ -2,6 +2,12 @@
 
 Use the System Configuration Service \(SCSD\) to set the BMCs credentials to unique values, or set them all to the same value. Redfish BMCs get installed into the system with default credentials. Once the machine is shipped, the Redfish credentials must be changed on all BMCs. This is done using System Configuration Service \(SCSD\) through the Cray CLI.
 
+**Important:** When changing credentials they may exist a device specific procedure that should be perfered over this one. List of device specific credential changing procedures:
```
Comment (nrockershousen) [inline operations/system_configuration_service/Set_BMC_Credentials.md:5]: Change 

"When changing credentials they may exist a device specific procedure that should be perfered over this one. List of device specific credential changing procedures:"

to

"If the credentials for other devices need to be changed, refer to the following device-specific credential changing procedures:"

File: operations/system_configuration_service/Set_BMC_Credentials.md [hunk 21]
```diff
@@ -2,6 +2,12 @@
...
+- To change liquid-cooled BMC credentials, refer to [Change Cray EX Liquid-Cooled Cabinet Global Default Password](../security_and_authentication/Change_EX_Liquid-Cooled_Cabinet_Global_Default_Password.md)
```
Comment (nrockershousen) [inline operations/system_configuration_service/Set_BMC_Credentials.md:6]: Missing period

File: operations/security_and_authentication/Update_default_ServerTech_PDU_credentials_used_by_the_Redfish_Translation_Service.md [hunk 22]
```diff
@@ -0,0 +1,198 @@
...
+        name: rts
+    spec:
+        charts:
+        - name: cray-hms-rts
+          version: $RTS_VERSION
+          namespace: services
+    EOF
+    ```
+
+2.  Merge customizations.yaml with rts-manifest.yaml
```
Comment (nrockershousen) [inline operations/security_and_authentication/Update_default_ServerTech_PDU_credentials_used_by_the_Redfish_Translation_Service.md:153]: Missing period

Comment (nrockershousen) [reply operations/security_and_authentication/Update_default_ServerTech_PDU_credentials_used_by_the_Redfish_Translation_Service.md:153]: Same comment as above about enclosing file names with backticks

File: operations/security_and_authentication/Update_default_ServerTech_PDU_credentials_used_by_the_Redfish_Translation_Service.md [hunk 23]
```diff
@@ -0,0 +1,198 @@
...
+
+The Redfish Translation Service provides a Redfish interface that CSM Services (HSM, and CAPMC) can use interact with ServerTech PDUs which do not natively support Redfish. 
+
+There are 2 sets of default credentials that are required for RTS to function:
+1. The default credentials to use when new ServerTech PDUs are discovered in the system.
+2. The global default credential that RTS uses for its Redfish interface with other CSM services.
+
+**Important:**: After this procedure is completed **going forward all future ServerTech PDUs** added to the system will be assumed to be already configured with the new global default credential when getting added to the system.
+
+**Note:** This procedure will not change the credentials on existing ServerTech PDUs in a system. To change the credential on existing air-cooled hardware follow the [Change credentials on ServerTech PDUs](Change_credentials_on_ServerTech_PDUs.md) procedure. However, this procedure will update the global default credential that RTS uses for its Redfish interface to other CSM services.
```
Comment (nrockershousen) [inline operations/security_and_authentication/Update_default_ServerTech_PDU_credentials_used_by_the_Redfish_Translation_Service.md:13]: - **Note**:  ->  **NOTE:**
- Add comma after "hardware" in the second sentence

File: operations/security_and_authentication/Update_default_ServerTech_PDU_credentials_used_by_the_Redfish_Translation_Service.md [hunk 24]
```diff
@@ -0,0 +1,198 @@
+# Update default ServerTech PDU credentials used by the Redfish Translation Service
```
Comment (nrockershousen) [inline operations/security_and_authentication/Update_default_ServerTech_PDU_credentials_used_by_the_Redfish_Translation_Service.md:1]: Capitalization change: Update Default ServerTech PDU Credentials used by the Redfish Translation Service (RTS)"

File: operations/security_and_authentication/Update_default_ServerTech_PDU_credentials_used_by_the_Redfish_Translation_Service.md [hunk 25]
```diff
@@ -0,0 +1,198 @@
...
+
+This procedure updates the default credentials used by the Redfish Translation Service (RTS) for when new ServerTech PDUs are discovered in a system.
+
+The Redfish Translation Service provides a Redfish interface that CSM Services (HSM, and CAPMC) can use interact with ServerTech PDUs which do not natively support Redfish.
```
Comment (nrockershousen) [inline operations/security_and_authentication/Update_default_ServerTech_PDU_credentials_used_by_the_Redfish_Translation_Service.md:5]: - Change "The Redfish Translation Service" to RTS
- Change "that CSM Services (HSM, and CAPMC)" to "that the Hardware State Manager (HSM) and Cray Advanced Platform Monitoring and Control (CAPMC) services"

File: operations/security_and_authentication/Update_default_ServerTech_PDU_credentials_used_by_the_Redfish_Translation_Service.md [hunk 26]
```diff
@@ -0,0 +1,198 @@
+# Update default ServerTech PDU credentials used by the Redfish Translation Service
+
+This procedure updates the default credentials used by the Redfish Translation Service (RTS) for when new ServerTech PDUs are discovered in a system.
+
+The Redfish Translation Service provides a Redfish interface that CSM Services (HSM, and CAPMC) can use interact with ServerTech PDUs which do not natively support Redfish. 
+
+There are 2 sets of default credentials that are required for RTS to function:
```
Comment (nrockershousen) [inline operations/security_and_authentication/Update_default_ServerTech_PDU_credentials_used_by_the_Redfish_Translation_Service.md:7]: 2 -> two

**Review by nrockershousen — CHANGES_REQUESTED** [review]

File: operations/security_and_authentication/System_Security_and_Authentication.md (hunk 13 above)
Comment (rsjostrand-hpe) [reply operations/security_and_authentication/System_Security_and_Authentication.md:38]: Just to be clear, this entire block can be remove: https://github.com/Cray-HPE/docs-csm/blob/0efffe6f1c88cc3bee73a4196e4369be6e75101a/operations/security_and_authentication/System_Security_and_Authentication.md?plain=1#L29-L76

Comment (nrockershousen) [reply operations/security_and_authentication/System_Security_and_Authentication.md:38]: Yes, that is correct.

File: operations/security_and_authentication/Change_Air-Cooled_Node_BMC_Credentials.md (hunk 1 above)
Comment (rsjostrand-hpe) [reply operations/security_and_authentication/Change_Air-Cooled_Node_BMC_Credentials.md]: Should be resolved fixed now.

File: operations/security_and_authentication/Change_Air-Cooled_Node_BMC_Credentials.md (hunk 2 above)
Comment (rsjostrand-hpe) [reply operations/security_and_authentication/Change_Air-Cooled_Node_BMC_Credentials.md]: Should be resolved fixed now.

File: operations/security_and_authentication/Change_SMNP_credentials_on_Leaf_switches.md (hunk 3 above)
Comment (rsjostrand-hpe) [reply operations/security_and_authentication/Change_SMNP_credentials_on_Leaf_switches.md]: Should be resolved fixed now.

**Comment by rsjostrand-hpe** [comment]
I believe I have addresses all of @nrockershousen comments.

File: operations/security_and_authentication/Change_SMNP_credentials_on_Leaf_switches.md (hunk 4 above)
Comment (rsjostrand-hpe) [reply operations/security_and_authentication/Change_SMNP_credentials_on_Leaf_switches.md]: Should be resolved fixed now.

Comment (rsjostrand-hpe) [reply operations/security_and_authentication/Change_SMNP_credentials_on_Leaf_switches.md]: Should be resolved fixed now.

File: operations/security_and_authentication/Change_SMNP_credentials_on_Leaf_switches.md (hunk 5 above)
Comment (rsjostrand-hpe) [reply operations/security_and_authentication/Change_SMNP_credentials_on_Leaf_switches.md]: Should be resolved fixed now.

File: operations/security_and_authentication/Change_SMNP_credentials_on_Leaf_switches.md (hunk 6 above)
Comment (rsjostrand-hpe) [reply operations/security_and_authentication/Change_SMNP_credentials_on_Leaf_switches.md]: Should be resolved fixed now.

File: operations/security_and_authentication/Change_SMNP_credentials_on_Leaf_switches.md (hunk 7 above)
Comment (rsjostrand-hpe) [reply operations/security_and_authentication/Change_SMNP_credentials_on_Leaf_switches.md]: Should be resolved fixed now.

File: operations/security_and_authentication/Change_SMNP_credentials_on_Leaf_switches.md (hunk 8 above)
Comment (rsjostrand-hpe) [reply operations/security_and_authentication/Change_SMNP_credentials_on_Leaf_switches.md]: Should be resolved fixed now.

File: operations/security_and_authentication/Change_credentials_on_ServerTech_PDUs.md (hunk 10 above)
Comment (rsjostrand-hpe) [reply operations/security_and_authentication/Change_credentials_on_ServerTech_PDUs.md]: Should be resolved fixed now.

File: operations/security_and_authentication/Change_credentials_on_ServerTech_PDUs.md (hunk 11 above)
Comment (rsjostrand-hpe) [reply operations/security_and_authentication/Change_credentials_on_ServerTech_PDUs.md]: Should be resolved fixed now.

Comment (rsjostrand-hpe) [reply operations/security_and_authentication/Change_credentials_on_ServerTech_PDUs.md]: Should be resolved fixed now.

File: operations/security_and_authentication/Change_credentials_on_ServerTech_PDUs.md (hunk 12 above)
Comment (rsjostrand-hpe) [reply operations/security_and_authentication/Change_credentials_on_ServerTech_PDUs.md]: Should be resolved fixed now.

File: operations/security_and_authentication/System_Security_and_Authentication.md (hunk 13 above)
Comment (rsjostrand-hpe) [reply operations/security_and_authentication/System_Security_and_Authentication.md]: Should be resolved fixed now.

File: operations/security_and_authentication/Update_default_Air-Cooled_BMC_and_Leaf_switch_SNMP_credentials.md (hunk 15 above)
Comment (rsjostrand-hpe) [reply operations/security_and_authentication/Update_default_Air-Cooled_BMC_and_Leaf_switch_SNMP_credentials.md]: Should be resolved fixed now.

File: operations/security_and_authentication/Update_default_Air-Cooled_BMC_and_Leaf_switch_SNMP_credentials.md (hunk 14 above)
Comment (rsjostrand-hpe) [reply operations/security_and_authentication/Update_default_Air-Cooled_BMC_and_Leaf_switch_SNMP_credentials.md]: Should be resolved fixed now.

File: operations/security_and_authentication/Update_default_Air-Cooled_BMC_and_Leaf_switch_SNMP_credentials.md (hunk 16 above)
Comment (rsjostrand-hpe) [reply operations/security_and_authentication/Update_default_Air-Cooled_BMC_and_Leaf_switch_SNMP_credentials.md]: Should be resolved fixed now.

File: operations/security_and_authentication/Update_default_Air-Cooled_BMC_and_Leaf_switch_SNMP_credentials.md (hunk 18 above)
Comment (rsjostrand-hpe) [reply operations/security_and_authentication/Update_default_Air-Cooled_BMC_and_Leaf_switch_SNMP_credentials.md]: Should be resolved fixed now.

File: operations/system_configuration_service/Set_BMC_Credentials.md (hunk 20 above)
Comment (rsjostrand-hpe) [reply operations/system_configuration_service/Set_BMC_Credentials.md]: Should be resolved fixed now.

File: operations/security_and_authentication/Change_credentials_on_ServerTech_PDUs.md (hunk 19 above)
Comment (rsjostrand-hpe) [reply operations/security_and_authentication/Change_credentials_on_ServerTech_PDUs.md]: Should be resolved fixed now.

File: operations/system_configuration_service/Set_BMC_Credentials.md (hunk 21 above)
Comment (rsjostrand-hpe) [reply operations/system_configuration_service/Set_BMC_Credentials.md]: Should be resolved fixed now.

File: operations/security_and_authentication/Update_default_ServerTech_PDU_credentials_used_by_the_Redfish_Translation_Service.md (hunk 22 above)
Comment (rsjostrand-hpe) [reply operations/security_and_authentication/Update_default_ServerTech_PDU_credentials_used_by_the_Redfish_Translation_Service.md]: Should be resolved fixed now.

**Review by nrockershousen — APPROVED** [review]

**Pull request merged**